### DISCUSSION TITLE
error connecting or unexpected enhanced byte 2

### DISCUSSION THREAD

**Discussion by samezrp** [john30/ebusd#799] [Q&A]
Hi,
I would like to ask for some help.
I have HW from Daniel Kucera's store.
When I try to acquire any signal running this command:

` ebusd -f -c /tmp --logareas bus --loglevel info --lograwdata=bytes -d enh:192.168.1.124:3333`

I have following error:
```
2023-01-06 14:20:04.572 [bus notice] device status: resetting
2023-01-06 14:20:04.572 [bus notice] bus started with own address 31/36
2023-01-06 14:20:04.583 [bus notice] device status: reset, supports info
2023-01-06 14:20:05.099 [bus error] device status: unexpected enhanced byte 2
2023-01-06 14:20:05.143 [bus error] device status: unexpected enhanced byte 2
2023-01-06 14:20:05.188 [bus error] device status: unexpected enhanced byte 2

```
I tried to adjust potentiometer according to wiki (with on and off diode, I could not find any oscilloscope instruction), sometimes data changes, but for a while, nothing stable.
I also tried to use tool to get any raw data:

`ebusctl -p 8888 -s 192.168.1.124 scan full`

but in this case I get error:

`error connecting to 192.168.1.124:8888`

Could you please point me what else can I check?

**Comment by danielkucera** [comment]
Remove `enh:`  . My adapter doesn't support enhanced protocol (yet).

**Reply by danielkucera** [reply]
Now it does but on a different port, see the readme:
https://github.com/danielkucera/esp-arduino-ebus?tab=readme-ov-file#quickstart

**Reply by step21** [reply]
Thanks! I mostly now have to figure out a nice way to process my data for home assistant, as my system seems to only give data as "Status01", "Status02", and "CirPumpState". Where the last is a simple state, but the others combine several values with ";", and value names only as a comment.

**Comment by nuki47** [comment]
Have you found a solution? because I also get

`2023-01-28 21:54:19.407 [bus error] device status: missing enhanced byte 2
2023-01-28 21:54:19.436 [bus error] device status: unexpected enhanced byte 2`

**Comment by samezrp** [comment]
You cannot use "enh" prefix in the command line as Daniel said.
Wiki for eBus is not clear for newbies, you can use just IP without any prefixes or so.

**Reply by step21** [reply]
thanks for this. I had the same issue also with ens: prefix, removing it helped.
